src/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -439,6 +440,10 @@ public void onFocusChange(View v, boolean hasFocus) {
         }
     }
 
+    public boolean isCheckedItemsZero(){
```

Comment:
The name for this method doesn't fit with what you are doing inside since you are verifying whether there's one file checked or not, is that correct?

Reply:
@davigonz I will make the function name change.